operations/experimental/disable-logs-in-firehose.yml

```diff
@@ -0,0 +1,8 @@
+---
+- type: replace
+  path: /addons/name=loggregator_agent/jobs/name=loggregator_agent/properties/disable_logs?
+  value: true
+
+- type: replace
+  path: /addons/name=loggregator_agent/jobs/name=loggregator_agent/properties/disable_logs?
```

Comment:
this appears to be exactly the same as the block above

Reply:
indeed, that might be a mistake.